test/token/token.go

```diff
@@ -123,33 +119,3 @@ func GenerateTokenWithClaims(claims map[string]interface{}) string {
 	}
 	return tokenStr
 }
-
-func ContextWithRequest(ctx context.Context) context.Context {
-	if ctx == nil {
-		ctx = context.Background()
-	}
-	u := &url.URL{
-		Scheme: "https",
-		Host:   "auth.openshift.io",
-	}
-	rw := httptest.NewRecorder()
-	req, err := http.NewRequest("GET", u.String(), nil)
-	if err != nil {
-		panic(errors.WithStack(err))
-	}
-	return goa.NewContext(goa.WithAction(ctx, "Test"), rw, req, url.Values{})
-}
-
-func ContextWithTokenAndRequestID() (context.Context, string, string, string) {
-	identityID := uuid.NewV4().String()
-	ctx, ctxToken := EmbedTokenInContext(identityID, uuid.NewV4().String())
-	ctx = tokencontext.ContextWithTokenManager(ctx, TokenManager)
-	reqID := uuid.NewV4().String()
-	ctx = client.SetContextRequestID(ctx, reqID)
-
-	return ctx, identityID, ctxToken, reqID
-}
```

Comment:
so this `ContextWithTokenAndRequestID()` is removed .. it's not useful ??

Reply:
right, let me restore it: it's not used in `common`, but it has duplicates in other repos, that may use this one instead in the future.